.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.18.x, 1.19.x, 1.20.x, 1.21.x]
+        go-version: [1.20.x, 1.21.x, 1.22.x, 1.23.x]
```

Comment:
Only the latest two versions are supported by Go, so I wonder if we could drop the rest.